skyreader/result.py

```diff
@@ -238,13 +239,14 @@ def isclose_nside(self,
         """Get whether the two nside's pixels are all "close"."""
         # zip-iterate each pixel-data
         nside_diffs = []
+        common_fields = tuple(_ for _ in self.PIXEL_TYPES[1].names if _ in set(self.pixel_fields).intersection(other.pixel_fields))
```

Comment:
Can you add a comment that the generator is used here to sort the values in the expected ordering? If ordering is unnecessary, I'd suggest an intersection of 3 sets.

Also, can you name this `fields_to_compare` or something similar? That's more to the point.